inst/extdata/mass-balance-settings.json

```diff
@@ -0,0 +1,100 @@
+{
+    "MassBalancePlots": [
+        {
+            "Name": "Name of mass balance sub-section (optional)",
+            "Molecules": [
+                "Aciclovir"
+            ],
+            "Groupings": [
+                {
+                    "Name": "Intracellular",
+                    "Include": [
+                        "Organism|**|Intracellular|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
```

Comment:
It is not wrong, however the Exclude section does not make much sense here, because there is no **Intracellular** compartment in the lumen.

Reply:
Then, I'll remove